<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TexturePropertiesView</class>
 <widget class="QDockWidget" name="TexturePropertiesView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>403</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>196</width>
    <height>403</height>
   </size>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>   Texture Properties</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QToolButton" name="addButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../shaderlab.qrc">
           <normaloff>:/ico/img/Add-icon.png</normaloff>:/ico/img/Add-icon.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QToolButton" name="removeButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../shaderlab.qrc">
           <normaloff>:/ico/delete</normaloff>:/ico/delete</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="loadButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../shaderlab.qrc">
           <normaloff>:/ico/img/exchange.png</normaloff>:/ico/img/exchange.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="add3DButton">
         <property name="text">
          <string>3D</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="textureList"/>
       </item>
       <item>
        <widget class="QLabel" name="textureName">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="imageContainer">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>160</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#imageContainer{
border: 1px solid black;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../shaderlab.qrc">:/ico/noImage</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../shaderlab.qrc"/>
 </resources>
 <connections/>
</ui>
